tests/iam_policies/user-role-rp.cfn.yaml

```diff
@@ -0,0 +1,796 @@
+AWSTemplateFormatVersion: '2010-09-09'
+
```

Comment:
I suppose `user-role-rp.cfn.yaml` is the copy of [user-role.cfn.yaml](https://github.com/aws/aws-parallelcluster/blob/develop/tests/iam_policies/user-role.cfn.yaml) with the only difference that in `user-role-rp.cfn.yaml` some permissions are restricted to resources (roles and policies) with the expected resource prefix used by the integration tests.

If this is the case I suggest to avoid such duplication because every change to the pcluster user role should then be reflected in two files that must be kept aligned.

What about reading the original yaml file within the test and apply the resource prefix changes at runtime?

Reply:
I had to make changes by adding a lot of different permissions I the policies as well as the permission boundary as well as removed the Policies related to Image Builder. If I keep these Image Builder policies my File goes over 50Kb and I wont be able to run this template and would have to upload the template through S3 bucket for my Stack to be created.

Reply:
Will put this changes in another PR as its non-blocking improvement.